topydo.conf

```diff
@@ -6,6 +6,7 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+; backup_count              = 5
```

Comment:
I think it should be uncommented. Like this, the default value (5) will silently kick in and make backups nevertheless, so it's better to be explicit about it.